libs/s25main/ingameWindows/iwObservate.cpp

```diff
@@ -172,59 +145,13 @@ void iwObservate::Draw_()
 
         view->Draw(road, parentView.GetSelectedPt(), false);
         // Draw indicator for center point
-        if(!followMovableId)
+        if(!following && !lastValid)
```

Comment:
This is the only place I can think of, where the race condition *could* be visible for a single frame. I don't really care if the indicator is drawn incorrectly for one frame once in a blue moon. Do you?

Reply:
Yeah I think it's fine too